hypha/apply/todo/options.py

```diff
@@ -62,23 +62,23 @@
     # :todo: actions for multiple stages of submission
     SUBMISSION_DRAFT: {
         "text": _(
-            'A Submission draft [<span class="truncate inline-block max-w-32 align-bottom ">{related.title}</span>]({link} "{related.title}") is waiting to be submitted'
+            '<span class="text-xs">{related.fund_name} #{related.application_id}</span><br>A draft application <span class="truncate inline-block max-w-32 align-bottom ">"{related.title}")</span> is waiting to be submitted'
```

Comment:
`")</span>` This close parenthesis is a left over it looks like.

Reply:
yep